Dockerfile

```diff
@@ -1,18 +1,13 @@
-FROM debian:buster
-
-ARG CHAIN_VERSION
-ENV _CHAIN_VERSION=${CHAIN_VERSION}
-RUN echo "DataHighway chain version ${_CHAIN_VERSION}"
-
+# build stage
+FROM rust as builder
+# create a project folder
 WORKDIR /dhx
 
-# FIXME - only copy necessary files to reduce size of image, and try using intermediate stages again
 COPY . .
 
 RUN apt-get update && apt-get install -y build-essential wget cmake pkg-config libssl-dev \
     openssl git gcc clang libclang-dev curl vim unzip screen bash \
     && curl https://getsubstrate.io -sSf | bash -s -- --fast \
-    && . ~/.cargo/env \
```

Comment:
i think we need this. we even added it to https://getsubstrate.io/

Reply:
this was erroring for me and after removing that line the build went thru fine, so its left out